rclcpp/test/test_parameter_events_subscriber.cpp

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) 2019 Intel Corporation
```

Comment:
Can you follow the copyright statement style that we use? i.e. without the `(c)`:

https://github.com/ros2/rclcpp/blob/64c0f86f14dfe2337816079c4d5991268aeb0635/rclcpp/include/rclcpp/any_executable.hpp#L1

Reply:
This seems to still be using `(c)`...